apps/dc_tools/odc/apps/dc_tools/sqs_to_dc.py

```diff
@@ -268,7 +280,7 @@ def queue_to_odc(
                         )
 
             # Index the dataset
-            do_index_update_dataset(metadata, uri, dc, doc2ds, update, allow_unsafe)
+            do_index_update_dataset(metadata, uri, dc, doc2ds, update, update_if_exists, allow_unsafe)
```

Comment:
there are now quite a few flags here, better to use named parameters rather than rely on order being correct.
`update=update, update_if_exists=update_if_exists, ...`